---
layout: archive
title: "Publications"
permalink: /publications_Xiangtao/ 
author_profile: true
---


1. G. Hao, Y. Fan, Z. Yu, Y. Su, H. Zhu, F. Wang, X. Chen, Y. Yang, G. Wang, K. Wong, **X. Li\***, Topological Identification and Interpretation for Single-cell Epigenetic Regulation Elucidation in Multi-tasks using scAGDE, **Nature Communications**, 2025.
 
3. H. Zhu, Y. Yang, Y. Wang, F. Wang, Y. Huang, Y. Chang, K. Wong\*, **X. Li\***, Dynamic characterization and interpretation for protein–RNA interactions across diverse cellular conditions using HDRNet, **Nature Communications**, 2023. (IF= 17.694, Q1)

1. Z. Yu, Y. Su, Y. Lu, F. Wang, S. Zhang, Y. Chang, K. Wong\*, **X. Li\***, Topological Identification and Interpretation for Single-cell Gene Regulation Elucidation across Multiple Platforms using scMGCA, **Nature Communications**, 2023. (IF= 17.694, Q1)


3. Y. Su, Z. Yu, Y. Yang, K. Wong, **X. Li\***, Distribution-agnostic Deep Learning Enables Accurate Single‐Cell Data Recovery and Transcriptional Regulation Interpretation, **Advanced Science**, 2023. (IF= 17.521, Q1)

4. Y. Fan, Y. Wang, F. Wang, L. Huang, Y. Yang, K. Wong, **X. Li\***, Reliable Identification and Interpretation of Single-cell Molecular Heterogeneity and Transcriptional Regulation using Dynamic Ensemble Pruning, **Advanced Science**, 2023. (IF= 17.521, Q1)

5. W. Ma, F. Wang, Y. Fan, G. Hao, Y. Su, K. Wong, **X. Li\***, DeepNanoHi-C: Deep Learning Enables Accurate Single-Cell Nanopore Long-read Data Analysis and 3D Genome Interpretation,  **Nucleic Acids Research**, 2025, (IF = 16.7, Q1)

6. F. Wang, J. Lin, H. Alinejad-Rokny, W. MA, L. Meng, L. Huang, J. Yu, N. Chen, Y. Wang, Z. Yao, W. Xie, K. Wong\*, **X. Li\***, Unveiling multi-scale architectural features in single-cell Hi-C data using scCAFE, **Advanced Science**, 2023. (IF= 17.521, Q1)

7. Y. Yang, G. Li, K. Pan, W. Cao, Z. Zhang\*, **X. Li\***, Deciphering 3’ UTR Mediated Gene Regulation Using Interpretable Deep Representation Learning, **Advanced Science**, 2024. (IF= 17.521, Q1)

8. Z. Yu, Y. Yang, X. Chen, K. Wong, Z. Zhang, Y. Zhao\*, **X. Li\***, Accurate Spatial Heterogeneity Dissection and Gene Regulation Interpretation for Spatial Transcriptomics using Dual Graph Contrastive Learning, **Advanced Science**, 2024. (IF= 17.521, Q1)

2. Z. Zheng, J. Chen, X. Chen, L. Huang, W. Xie, Q. Lin, **X. Li\***, K. Wong\*, Enabling Single-cell Drug Response Annotations from Bulk RNA- seq using SCAD, **Advanced Science**, 2023. (IF= 17.521, Q1)

3. F. Wang, H. Alinejad-Rokny, J. Lin, T. Gao, X. Chen, L. Meng, **X. Li\***, K. Wong\*, A lightweight framework for chromatin loop detection at the Single‐Cell Level, **Advanced Science**, 2023. (IF= 17.521, Q1)

4. N. Chen, J. Yu, Z. Liu, L. Meng, **X. Li\***, K. Wong\*, Discovering DNA shape motifs with multiple DNA shape features: generalization, methods, and validation, **Nucleic Acids Research**, 2024, (IF = 14.9, Q1)

5. Y. Wang, H. Zhu, Y. Wang, Y. Yang, Y. Huang\*, J. Zhang\*, K. Wong,  **X. Li\***, EnrichRBP: an automated and interpretable computational platform for predicting and analyzing RNA-binding protein events, **Bioinformatics**, 2024. (**IF=6.931**,Q1)


6. B. Zhang, Z. Hou, K. Wong, H. Zhu\*, **X. Li\***, SOFB is a Comprehensive Ensemble Deep Learning approach for Elucidating and Characterizing Protein-Nucleic-Acid-Binding Residues, **Communications Biology**, 2024. (Q1)

7. Y. Zhang, Y. Zhuo, K. Wong, **X. Li\***, Unraveling Spatial Domain Characterization in Spatially Resolved Transcriptomics with Robust Graph Contrastive Clustering, **Bioinformatics**, 2024. (**IF=6.931**,Q1)

8. Q. Qi, Y. Wang, Y. Huang, Y. Fan \*, **X. Li\***, PredGCN: A Pruning-enabled Gene-Cell Net for Automatic Cell Annotation of Single Cell Transcriptome Data, **Bioinformatics**, 2024. (**IF=6.931**,Q1)

1. Y. Wang, Y. Zhu, S. Li, C. Bian, Y. Liang, K. Wong, **X. Li\***, scBGEDA: Deep Single-cell Clustering Analysis via a Dual Denoising Autoencoder with Bipartite Graph Ensemble Clustering, **Bioinformatics**, 2023. (**IF=6.931**,Q1)

2. P. Sun, S. Fan, S. Li, Y. Zhao, C. Lu\*, K. Wong, **X. Li\***, Automated Exploitation of Deep Learning for Cancer Patient Stratification across Multiple Types, **Bioinformatics**, 2023. (**IF=6.931**,Q1)

3. M. Toseef, O. O. Petinrin, F. Wang, S. Rahaman, Z. Liu, **X. Li\***, K. Wong\*, Deep transfer learning for clinical decision-making
based on high-throughput data: comprehensive survey with benchmark results, **Briefings in Bioinformatics**, 2023. (**IF=9.5**, Q1)

4. Z. Hou, Y. Yang. Z. Ma, K. Wong, **X. Li\***, Learning the Protein Language of Proteome-wide Protein-protein Binding Sites via Explainable Ensemble Deep Learning, **Communications Biology**, 2022. (Q1)

1. Y. Wang, Z. Hou, Y. Yang, K. Wong, **X. Li\***, Genome-wide Identification and Characterization of DNA Enhancers with a Stacked Multivariate Fusion Framework, **PLOS Computational Biology**, 2022. (Q1)

1. Y. Cheng, Y. Su, Z. Yu, Y. Liang, K. Wong, **X. Li\***, Unsupervised Deep Embedded Fusion Representation of Single-cell Transcriptomics, **Thirty-Seventh AAAI Conference on Artificial Intelligence (AAAI 2023)**, 2022. (Q1, **Oral**)

1. F. Wang, T. Gao, J. Lin, Z. Zheng, L. Huang, M. Toseef, **X. Li\***, K. Wong\*, GILoop: robust chromatin loop calling across multiple sequencing depths on Hi-C data, **iScience**, 2022. (**IF=6.107**, *Cell Press*)

1. L. Huang, J. Lin, R. Liu, Z. Zhang, L. Meng, X. Chen, **X. Li\***, K. Wong\*, CoaDTI: Multi-modal Co-attention based framework for drug-target interaction annotation, **Briefings in Bioinformatics**, 2022. (**IF=13.994**, Q1)

1. Y. Su, H. Zhu, K. Wong, Y. Chang, **X. Li\***, Hyperspectral Image Denoising via Weighted Multidirectional Low-rank Tensor Recovery, **IEEE Transactions on Cybernetics**, 2022. (**IF=19.118**,Q1) 

3. Y. Su, F. Wang, S. Zhang, Y. Liang, K. Wong, **X. Li\***, scWMC: Weighted Matrix Completion-based Imputation of scRNA-seq Data via Prior Subspace Information, **Bioinformatics**, 2022. (**IF=6.931**,Q1) 

1. F. Lu, Z. Yu, Y. Wang, Z. Ma, K. Wong, **X. Li\***, GMHCC: High-throughput Analysis of Biomolecular Data using Graph-based Multiple Hierarchical Consensus Clustering, **Bioinformatics**, 2022. (**IF=6.937**,Q1) 

1. M. Toseef, **X. Li\***, K. Wong\*, Reducing healthcare disparities using multiple multiethnic data distributions with fine-tuning of transfer learning, **Briefings in Bioinformatics**, 2022. (**IF=11.622**, Q1)
 
3. Y. Yang, Z. Hou, Y. Wang, H. Ma, P. Sun, Z. Ma, K. Wong, **X. Li\***, HCRNet: High-throughput circRNA-Binding Event Identification from CLIP-seq Data using Deep Temporal Convolutional Network, **Briefings in Bioinformatics**, 2022. (**IF=11.622**, Q1)

1. Z. Yu, Y. Lu, Y. Wang, F. Tang, K. Wong, **X. Li\***, ZINB-based Graph Embedding Autoencoder for Single-cell RNA-seq Interpretations, **Thirty-Sixth AAAI Conference on Artificial Intelligence (AAAI 2022)**, 2021. (Q1, **Oral**)

21. Y. Wang, Y. Yang, Z. Ma, K. Wong, **X. Li\***, EDCNN: Identification of Genome-Wide RNA-binding Proteins Using Evolutionary Deep Convolutional Neural Network, **Bioinformatics**, 2021. (**IF=6.937**, Q1)

3. L. Huang, J. Lin, **X. Li\***, L. Song, Z. Zheng, and K. Wong\*, EGFI: Drug-Drug Interaction Extraction and Generation with Fusion of Enriched Entity and Sentence Information, **Briefings in Bioinformatics**, 2021. (**IF=11.622**, Q1)

3. Y. Wang, C. Bian, K. Wong, **X. Li\***, S. Yang. Multiobjective Deep Clustering and Its Applications in Single-cell RNA-seq Data, **IEEE Transactions on Systems, Man, and Cybernetics: Systems**, 2021. (**IF=13.451**, Q1)

1. **X. Li**, S. Li, L. Huang, S. Zhang, K. Wong. High-throughput Single-cell RNA-seq Data Imputation and Characterization with Surrogate-assisted Automated Deep Learning, **Briefings in Bioinformatics**, 2021. (**IF=11.622**, Q1)

3. Z. Hou, Y. Yang, H. Li, K. Wong, **X. Li\***. iDeepSubMito: Identification of protein sub-mitochondrial localization with deep learning, **Briefings in Bioinformatics**, 2021. (**IF=11.622**, Q1)

5. Y. Wang,  **X. Li\***, K. Wong, Y. Chang, S. Yang. Evolutionary Multiobjective Clustering Algorithms with Ensemble for Patient Stratification, **IEEE Transactions on Cybernetics**, 2021. (**IF=11.448**, Q1)

7. Z. Yu, C. Bian, G. Liu, S. Zhang, K. Wong,  **X. Li\***. Elucidating Transcriptomic Profiles from Single-cell RNA sequencing Data using Nature-Inspired Compressed Sensing, **Briefings in Bioinformatics**, 2021. (**IF=11.622**, Q1)

3. **X. Li**, S. Zhang, K. Wong. Deep Embedded Clustering with Multiple Objectives on scRNA-seq Data, **Briefings in Bioinformatics**, 2021. (**IF=11.622**, Q1)

3. Y. Yang, S. Li, Y. Wang, K. Wong, **X. Li\***. Identification of Haploinsufficient Genes from Epigenomic Data using Deep Forest, **Briefings in Bioinformatics**, 2020. (**IF=11.622**, Q1)

23. Zhang S, **Li X**, Lin Q, Wong KC. Uncovering the key dimensions of high-throughput biomolecular data using deep learning,**Nucleic Acids Research**, 2020. (**IF=16.971**, Q1)

1. **X. Li**, S. Li, Y. Wang, S. Zhang, K. Wong. Identification of Pan-cancer Ras Pathway Activation with Deep Learning, **Briefings in Bioinformatics**, 2020. (**IF=11.622**, Q1)

2. Y. Yang, Z. Hou, Z. Ma, **X. Li\***, K. Wong\*, iCircRBP-DHN: identification of circRNA-RBP interaction sites using deep hierarchical network, **Briefings in Bioinformatics**, 2020. (**IF=11.622**, Q1)

22. K.C. Wong J. Lin, **X. Li**, Q. Lin; C. Liang, Y. Song, Hterodimeric DNA Motif Synthesis and Validations, **Nucleic Acids Research**, 2019. (**IF=16.971**, Q1)

24. **X. Li**, S. Zhang, K. Wong. Multiobjective Genome-Wide RNA-Binding Event Identification from CLIP-seq Data, **IEEE Transactions on Cybernetics**, doi.10.1109/TCYB.2019.2960515, 2019. (**IF=11.448**, Q1)

4.  **X. Li**, K. Wong. Evolutionary Multi-objective Clustering and Its Applications to Patient Stratification, **IEEE Transactions on Cybernetics**, doi.10.1109/TCYB.2018.2817480, 2018. (**IF=11.448**, Q1)

5. Zhang S, **Li X**, Lin Q and Wong KC. Nature-Inspired Compressed Sensing for Transcriptomic Profiling From Random Composite Measurements.**IEEE Transactions on Cybernetics**, 2019.(**IF=11.448**, Q1)

6. **X. Li**, S. Zhang, K. Wong. Single-cell RNA-seq Interpretations using Evolutionary Multiobjective Ensemble Pruning, **Bioinformatics**, 2019, doi. 10.1093/bioinformatics/bty1056. (**IF=6.937**, Q1)

7. Zhang S,**Li X**, Lin Q, Wong KC. Synergizing CRISPR/Cas9 Off-Target Predictions for Ensemble Insights and Practical Applications. **Bioinformatics**, 2019, 35 (7): 1108-1115. (**IF=6.937**,Q1) 

11. **X. Li**, K. Wong. Multiobjective Patient Stratification using Evolutionary Multiobjective Optimization. **IEEE Journal of Biomedical and Health Informatics**, doi.10.1109/JBHI.2017.2769711, 2017. (Q1)

14. **X. Li**, M. Li, Multiobjective Local Search algorithm based decomposition for Multiobjective Permutation Flowshop Scheduling Problem, **IEEE Transactions on Engineering Management**, 2015, 62(4): 544-557.(**IF=6.146, Top journal for Engineering Management**)

12. **X. Li**, S. Ma, Multi-objective Discrete Artificial Bee Colony Algorithm for Multi-objective Permutation Flow Shop Scheduling Problem with Sequence Dependent Setup Times, **IEEE Transactions on Engineering Management**, 64(2)(2016): 149-165. (**IF=6.146, Top journal for Engineering Management**)

6. **X. Li**, S. Zhang, K. Wong. Evolving Transcriptomic Profiles from Single-cell RNA-seq Data using Nature-Inspired Multiobjective Optimization, **IEEE/ACM Transactions on Computational Biology and Bioinformatics**, doi. 10.1109/TCBB.2020.2971993, 2020.

7. Y. Wang, Q. Ma, K. Wong, **X. Li\***. Evolving Multiobjective Cancer Subtype Diagnosis from Cancer Gene Expression Data, **IEEE/ACM Transactions on Computational Biology and Bioinformatics**, doi.10.1109/TCBB.2020.2974953, 2020.

8. **X. Li**, K. Wong. Single-Cell RNA-seq Data Interpretation by Evolutionary Multiobjective Clustering, **IEEE/ACM Transactions on Computational Biology and Bioinformatics**, doi.10.1109/TCBB.2019.2906601, 2019.

9. **X. Li**, S. Zhang, K. Wong. Nature-Inspired Multiobjective Epistasis Elucidation from Genome-Wide Association Studies, **IEEE/ACM Transactions on Computational Biology and Bioinformatics**, doi. 10.1109/TCBB.2018.2849759, 2018.

10. **X. Li**, K. Wong. Elucidating Genome-Wide Protein-RNA Interactions using Differential Evolution, **IEEE/ACM Transactions on Computational Biology and Bioinformatics**, doi. 10.1109/TCBB.2017.2776224, 2017.


15. **X. Li**, K. Wong, A Comparative Study for Identifying the Chromosome-Wide Spatial Clusters from High-Throughput Chromatin Conformation Capture data, **IEEE/ACM Transactions on Computational Biology and Bioinformatics**, doi: 10.1109/TCBB.2017.2684800, 2017.

23. Wong, K. C., Yan, S., Lin, Q., **X. Li**, & Peng, C. Deleterious Non-Synonymous Single Nucleotide Polymorphism Predictions on Human Transcription Factors. **IEEE/ACM transactions on computational biology and bioinformatics**. 2019.

13. **X. Li**, M. Yin, Multiobjective Binary Biogeography based Optimization based Feature Selection for Gene Expression Data, **IEEE Transactions on NanoBioscience**, 12 (4) (2013): 343- 353.

16. **X. Li**, S. Ma, K. Wong, Evolving Spatial Clusters of Genomic Regions from High-Throughput Chromatin Conformation Capture data, **IEEE Transactions on NanoBioscience**,16(6) (2017), 400-407.

17. Y. Wang, B. Liu, Z. Ma, K. Wong, **X. Li\***, Nature-Inspired Multiobjective Cancer Subtype Diagnosis, **IEEE Journal of Translational Engineering in Health and Medicine**, Accepted, 2019.


18. **X. Li**, M. Yin, Modified Cuckoo search algorithm with self adaptive parameter method, **Information Sciences**, 2015, 298:80-97. (**IF=6.795**, Q1)

19. Y. Wang, Z. Ma, K. Wong, **X. Li\***, Nature-Inspired Multiobjective Patient Stratification from Cancer Gene Expression Data, **Information Sciences**, 2020, Accepted. (**IF=6.795**, Q1)

20. T. Ma, Y. Wang\*, **X. Li\***, Convex Combination Multiple Populations Competitive Swarm Optimization for moving target search using UAVs, **Information Sciences**, 2023, Accepted. (**IF=8.233**, Q1)

21. X. Wang, Y. Wang, K. Wong, **X. Li\***, A Self-adaptive Weighted Differential Evolution Approach for Large-scale Feature Selection, *Knowledge-based Systems*, 2021. (**IF=8.038**, Q1)

23. H. Zhu, Y. Wang\*, **X. Li\***, UCAV Path Planning for Avoiding Obstacles using Cooperative Co-evolution Spider Monkey Optimization,*Knowledge-based Systems*, 2022. (**IF=8.038**, Q1)

24. C. Bian, H. Gao, Y. Wang\*, K. Wong, **X. Li\***, scEFSC: Accurate Single-cell RNA-seq Data Analysis via Ensemble Consensus Clustering Based on Multiple Feature Selections, *Computational and Structural Biotechnology Journal*, 2022. (**IF=7.271**)

27. **X. Li**, M. Yin, Parameter estimation for chaotic systems by hybrid differential evolution algorithm and artificial bee colony algorithm, *Nonlinear Dynamics*, 77(2014) 61-71. (Q1)

29. **X. Li**, J.Y. Wang, J.P. Zhou, M. Yin, A perturb biogeography based optimization with mutation for global numerical optimization, *Applied Mathematics and Computation*, 2 011,218(2):598-609. (Q1)

20. **X. Li**, S. Ma, G. Yang, Synthesis of Difference Patterns for Monopulse Antennas by an Improved Cuckoo Search Algorithm, *IEEE Antennas and Wireless Propagation Letters*, 2016, doi: 10.1109/LAWP.2016.2640998.

24. **X. Li**, M. Yin, Modified Differential Evolution with Self-adaptive Parameters Method, *Journal of Combinatorial Optimization*, 2016, 31(2):546-576.

28. **X. Li**, M. Yin, Multi-operator based Biogeography based Optimization with Mutation for Global Numerical Optimization, *Computers and Mathematics with Applications*, 64(9)(2012) 2833-2844.

30. **X. Li**, M. Yin, An Opposition-Based Differential Evolution Algorithm for Permutation Flow shop Scheduling Based on Diversity Measure, *Advances in Engineering Software*. 55(2013)10-31.

31. **X. Li**, M. Yin, A Hybrid Cuckoo Search via Lévy Flights for Permutation Flow shop Scheduling Problem. *International Journal of Production Research*, 51(16)(2013):4732-4754.

36. Ka-Chun, Wong, Jiao, Zhang, Shankai, Yan, **Xiangtao, Li**, Qiuzhen, Lin, Sam, Kwong & Cheng, Liang, DNA Sequencing Technologies: Sequencing Data Protocols and Bioinformatics Tools. *ACM Computing Surveys*. (Q1)
